tools/ipam-autopilot/infrastructure/main.tf

```diff
@@ -0,0 +1,169 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+locals {
+  apis = ["iam.googleapis.com", "run.googleapis.com", "compute.googleapis.com", "cloudasset.googleapis.com", "sql-component.googleapis.com", "cloudapis.googleapis.com", "sqladmin.googleapis.com", "secretmanager.googleapis.com",  "artifactregistry.googleapis.com"]
```

Comment:
I believe in the terraform style guide each value has to be in it's own line. Can you run a terraform fmt against the folder that contains the .tf files and append to the next commit please ?

Reply:
I ran terraform fmt, it placed all of them in a single line.